package.json

```diff
@@ -30,7 +30,7 @@
   "license": "MIT",
   "private": true,
   "devDependencies": {
-    "@metaplex-foundation/solita": "0.0.2",
+    "@metaplex-foundation/solita": "0.1.0",
```

Comment:
Just remove solita from here (it should be a dev dep of each contract) ... it was a mistake to add it in the first place.